README.md

```diff
@@ -1 +1,222 @@
+
+- npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+or latest version:
+- npm install --save-dev stylelint stylelint-scss stylelint-config-standard stylelint-csstree-validator
+
+- npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+
+or latest version:
+- npm install --save-dev eslint  eslint-config-airbnb-base  eslint-plugin-import  babel-eslint
+
+## Requirements
+
+- Linters configuration.
+
+Clone the repository to get start with project, then make sure to install dependencies in the linters file located in the [linter](https://github.com/Bateyjosue/linters-html-css/blob/main/.github/workflows/linters.yml) file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Run tests
+
+- to test and check the html file/s is error-free run: 
+npx hint . 
+
+- to fix errors run:
+npx hint . -f
+
+- to test and check the css file/s is error-free run: 
+npx stylelint "**/*.{css,scss}"
+- to fix errors run:
+npx stylelint "**/*.{css,scss}" --fix
+
+- to test and check the js file/s is error-free run: 
+npx eslint .
+
+- to fix errors run:
+npx eslint . --fix
+
+### Deployment
+
+Check for the tests when you generate a pull request and fix the errors if any.
+For stylelint error run:<code>sudo npx stylelint "\*_/_.{css,scss}" --fix</code> and it will the fix style issues automatically.
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Jugoslav Achkoski**
+
+- GitHub: [jugosack](https://github.com/jugosack)
+- Twitter: [Jugoslav Achkoski](https://twitter.com/Jugosla22401325)
+- LinkedIn: [Jugoslav Achkoski](https://www.linkedin.com/in/jugoslav-achkoski-3a074021/?originalSubdomain=mk)
+
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add local storage to save todos
+- [ ] Add functionality to add and remove todos
+
+
+  <!-- CONTRIBUTING -->
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page
+
+
+<!-- SUPPORT -->
+
+## 👋 Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right"><a href="#readme-top">(back to top)</a></p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+- [Microverse Team](https://www.microverse.org/).
+
+I would like to thank Microverse for the information provided to build this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **Can I use the project for any purpose?**
+
+  - Yes, you can use this files for anything you need.
+
+- **Is the information saved in any database?**
+
+  - No, all data is saved in Local Storage.
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/microverseinc/readme-template/blob/master/MIT.md) licensed.
```

Comment:
- [ ] when click on MIT link I find MIT License it's only a placeholder please make sure to add your information so that it will be a valid license